common/pulp/common/constants.py

```diff
@@ -13,3 +13,9 @@
 TASK_FINAL_STATES = (TASK_STATES.SKIPPED, TASK_STATES.COMPLETED, TASK_STATES.FAILED,
                      TASK_STATES.CANCELED)
 TASK_INCOMPLETE_STATES = (TASK_STATES.WAITING, TASK_STATES.RUNNING)
+
+# The interval in seconds for which the Celery Process monitor thread sleeps between
+# checking for missing Celery processes.
+CELERY_CHECK_INTERVAL = 60
```

Comment:
I was even thinking if it shouldn't be included in settings instead of hard-coded constant.